resources/js/Pages/Admin/Rooms/UpdateRoomForm.vue

```diff
@@ -71,6 +71,19 @@
                     />
                 </div>
 
+             <div class="m-6">
+                <jet-label for="room_type" value="room_type" />
+                <select v-model="form.room_type" class="mt-1 block w-full" name="room_type" id="room_typeUpdate">
+                    <option value="lounge" :selected="form.room_type=='lounge'">Lounge</option> 
```

Comment:
Load these options from wherever they will be stored (see previous comment)

Reply:
loading these from backend will be a little complicated. basically need to update the controller to send a availableRoomTypes prop to frontend